modules/tinymce/src/core/main/ts/api/EditorManager.ts

```diff
@@ -270,10 +270,23 @@ const EditorManager: EditorManager = {
   },
 
   /**
-   * Overrides the default options for editor instances.
+   * Overrides the default options for editor instances. <code>overrideDefaults</code> replaces any of the defaults set by a previous call to the <code>overrideDefaults</code> function.
+   * <br /><br />
+   * When using the cloud, some of these defaults are required for the cloud-based editor to function.
+   * <br /><br />
+   * Therefore, when using <code>overrideDefaults</code> with the cloud-based editor, combine the previous defaults, <code>tinymce.defaultOptions</code>, with the new options that you want to integrate.
```

Comment:
Is _the cloud_ a generic reference to running TinyMCE on a remote system or is it a reference to the Tiny Cloud service?

If the latter, I’m inclined to argue for referencing it specifically: `s/cloud/Tiny Cloud/`.